src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -44,12 +43,14 @@ public Phone getPhone() {
         return phone;
     }
 
+
+
     public Email getEmail() {
         return email;
     }
 
-    public Address getAddress() {
-        return address;
+    public Telegram getTelegramId() {
```

Comment:
Would it be better to refactor this to `getTelegram` so that it's consistent with the `getTelegram` and `setTelegram` in `EditCommandParser` (or refactor the ones in `Edit` to `getTelegramId` and `setTelegramId`)?